application/src/test/java/org/togetherjava/tjbot/commands/tags/TagManageCommandTest.java

```diff
@@ -0,0 +1,393 @@
+package org.togetherjava.tjbot.commands.tags;
+
+import net.dv8tion.jda.api.MessageBuilder;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.requests.ErrorResponse;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.togetherjava.tjbot.commands.SlashCommand;
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.db.Database;
+import org.togetherjava.tjbot.db.generated.tables.Tags;
+import org.togetherjava.tjbot.jda.JdaTester;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.AdditionalMatchers.aryEq;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+final class TagManageCommandTest {
```

Comment:
Add a DisplayName to all tests in this class

Reply:
What if the test-method-name is already descriptive enough? I.e. if the DisplayName would just be a 1:1 copy of it?

I get that it makes sense to use it if u need more room to explain it, but thats not always the case, is it?

Reply:
As Tais said. The intention of the DisplayName is for a more human-readable label.

IDEs show it when running tests, coverage reports also use them as titles, and your build logs also use them as a title "I don't remember if this last one has to be toggled on or not"

Reply:
Thats fair. I tried to experiment a bit with the different conventions I know. So you see some more basic tests that have just the name of the method they test and then there are also more complex tests where each test only tests one specific scenario, using the `GIVEN, WHEN, THEN` syntax.

I agree on the `@DisplayName` bit, will add it.